tools/docker/image-build.sh

```diff
@@ -34,10 +36,11 @@ main() {
 
     while true; do
         case "$1" in
-            -v | --verbose)         VERBOSE=true; shift ;;
             -h | --help)            usage; exit 0; shift ;;
             -b | --base-image)      IMAGE="$2"; shift ; shift ;;
             -t | --tag)             TAG=":$2"; shift ; shift ;;
+            --no-builder)           BUILDER=false; shift ;;
+            --no-runner)            RUNNER=false; shift ;;
```

Comment:
Wouldn't it be nice to be able to specify the name of the image to build instead? (maybe the default could be to generate them all as it was before).
We have at least 2 other images (that are not yet built with this script), but I can't imagine having a `--no-<image>` option for all of them.

(nitpick)

Reply:
You're right, it's a good idea, I'll make the change